cms/server/templates/contest/task_submissions.html

```diff
@@ -101,21 +101,41 @@ <h2 style="margin-bottom: 10px">{{ _("Submit a solution") }}</h2>
 {% end %}
 
 <div id="submit_solution" class="row">
-    <div class="span5">
+    <div class="span9">
         <form class="form-horizontal" enctype="multipart/form-data" action="{{ url_root }}/tasks/{{ encode_for_url(task.name) }}/submit" method="POST">
-            <fieldset>
+            <fieldset class="row-fluid">
 {% for idx, filename in enumerate(x.filename for x in task.submission_format) %}
                 <div class="control-group">
-                    <label class="control-label" for="input{{ idx }}">{{ filename.replace(".%l", "") }}</label>
-                    <div class="controls">
+                        <label class="control-label span2" for="input{{ idx }}">{{ filename.replace(".%l", "") }}</label>
+
+                    <div class="span4">
                         <input type="file" class="input-xlarge" id="input{{ idx }}" name="{{ filename }}"/>
+                        <!-- FIXME: figure out why we had div class="controls" wrapping this -->
+                    </div>
+
+                    <!-- FIXME: What does this even do when there are multiple inputs? -->
```

Comment:
I was referring to submitting more than one file, too -- CMS treats them equally (it just unzips the zip file and pretends you uploaded them all individually). It's definitely not likely to be a case in the AIO where we need you to submit 2 files but I guess it's possible for some communication-style problems? *shrugs*.

I'm not actually sure how User Tests behave -- if the problem expects one cpp file and you upload, say, 3 for the test, I have no idea what it actually does. In any case you likely want to duplicate your dropdown stuff for the UserTestHandler since they just copy-pasted the code from the regular SubmitHandler (>:C)